autoload/neomake.vim

```diff
@@ -84,29 +84,29 @@ function! neomake#MakeJob(maker) abort
     endif
 endfunction
 
-function! neomake#GetMaker(name, ...) abort
+function! neomake#GetMaker(maker_id, ...) abort
```

Comment:
I don't think this argument's name needed to change, since as far as I can tell it is the same thing as it was before.